src/app/core/services/advance-request.service.ts

```
@@ -116,30 +113,53 @@
+  getApproverAdvanceRequest(id: string): Observable<ExtendedAdvanceRequest> {
+    return this.getApproverAdvanceRequestRaw(id).pipe(
+      map((advanceRequestPlatform) => this.transformApproverAdvReq(advanceRequestPlatform))
+    );
   }
 
   @CacheBuster({
     cacheBusterNotifier: advanceRequestsCacheBuster$,
   })
-  delete(advanceRequestId: string): Observable<AdvanceRequests> {
-    return this.apiService.delete('/advance_requests/' + advanceRequestId);
+  delete(advanceRequestId: string): Observable<{}> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)